### DISCUSSION TITLE
Feature Request: Render Images in Edit Mode

### DISCUSSION THREAD

**Discussion by GraphicOwls** [notaapp/talk#302] [Feature Requests]
It would be great if images would render in edit more and span full width. A lot of times I have to reference image charts while writing for language learning, and it creates friction when we have to keep toggling preview. Just makes it hard to work that way.

Thanks!

**Comment by erusev** [comment]
Would you like all images to render at max width or would you prefer the images to work as they do now (as thumbnails) but have a way to make individual images full-width? Thanks!

**Reply by GraphicOwls** [reply]
I think having a way to choose and make images render at full width would be more versatile for most users. So, maybe that's the way to go. Along with the ability to tap/click to expand in a simple lightbox would also be ideal. Being able to see and do this in edit mode would be amazing 😉
